web/client/components/map/cesium/DrawMeasureSupport.jsx

```diff
@@ -624,6 +648,28 @@ function DrawMeasureSupport({
                 });
             }
             break;
+        case MeasureTypes.LENGTH:
+            tooltipLabelText = tooltips.start;
+            if (coordinates.length > 1) {
```

Comment:
Here we should use the geodesicCoordinates instead of cartesian coordinates to visualize and compute distances. We could pass directly from the onDrawStart, onDrawing and onDrawEnd an additional property called geodesicCoordinates so we are able to use it here